config/projects-to-test/openjdk-17-projects-to-test-on.config

```diff
@@ -5,4 +5,4 @@
 # This file has been created to alleviate confusion in the contribution
 # repository's projects-to-test-on.properties file; these projects
 # must be used with special configurations.
-openjdk17|git|https://github.com/openjdk/jdk17.git|master||
+openjdk17|git|https://github.com/openjdk/jdk17.git|master|/openjdk17/test/.*/MalformedDataRB_en.properties|
```

Comment:
please explain,
It is not clear why this was not required before.
if there is good reason , lets merge such changes sooner.

Reply:
Reasoning at https://github.com/checkstyle/contribution/pull/747#discussion_r1059674964